srcs/builtin/exit.c

```diff
@@ -0,0 +1,44 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   exit.c                                             :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: kitsuki <kitsuki@student.42tokyo.jp>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/05/21 14:36:33 by kitsuki           #+#    #+#             */
+/*   Updated: 2023/05/21 14:36:33 by kitsuki          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "_environ.h"
+#include "ft_is/ft_is.h"
+#include "ft_string/ft_string.h"
+#include <stdlib.h>
```

Comment:
23行目で`errno`を参照しているので、`errno.h`のincludeをお願いします。

```suggestion
#include <errno.h>
#include "_environ.h"
#include "ft_is/ft_is.h"
#include "ft_string/ft_string.h"
#include <stdlib.h>
```